src/main/java/org/mockito/verification/VerificationSucceededEvent.java

```diff
@@ -0,0 +1,26 @@
+package org.mockito.verification;
+
+public class VerificationSucceededEvent {
```

Comment:
@szczepiq Can you explain why you would like to expose the events as interfaces?  The events are pure value objects without any behaviour. So there no need to mock them. Also there will be only one implementation so the event classes can be made final to avoid subclassing.